doc/source/index.rst

```diff
@@ -1,159 +1,31 @@
-Ray
-===
+What is Ray?
+============
 
-.. raw:: html
+.. include:: ray-overview/basics.rst
 
-  <embed>
-    <a href="https://github.com/ray-project/ray"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/365986a132ccd6a44c23a9169022c0b5c890c387/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f7265645f6161303030302e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_red_aa0000.png"></a>
-  </embed>
+Getting Started with Ray
+------------------------
 
-.. image:: https://github.com/ray-project/ray/raw/master/doc/source/images/ray_header_logo.png
+Checkout :ref:`gentle-intro` to learn more about Ray or jump to :ref:`overview-libs` to learn
+more about the ecosystem of libraries built on top of Ray to enable things like distributed hyperparameter tuning,
+Reinforcement Learning, and distributed training.
 
-**Ray is a fast and simple framework for building and running distributed applications.**
```

Comment:
this is largely moved to `basics.rst` where I can inline it into several documents since it's relevant both on the home page as well as the gentle introduction.